index.html

```diff
@@ -195,7 +203,7 @@
                             if (timer.duration <= 0) {
                                 clearInterval(intervalId);
                                 countdown.textContent = "Time's up!";
-                                startStopButton.disabled = true;
+                                alarmIntervalId = setInterval(playAlarmSound, 1000);
```

Comment:
To have the beep sound continue until the Stop button is pressed, you can modify the playAlarmSound function and the event listener for the Start/Stop button. Here's the updated code: